templates/event.php

```diff
@@ -23,4 +30,34 @@
 			<button class="button is-primary" id="join-event">Attend Event</button>
 		</div>
 	</div>
+	<div class="event-details-stats">
+		<h2>Stats</h2>
+		<table>
+			<thead>
+			<tr>
+				<th scope="col">Locale</th>
+				<th scope="col">Translations created</th>
+				<th scope="col">Translations reviewed</th>
+				<th scope="col">Contributors</th>
+			</tr>
+			</thead>
+			<tbody>
+			<?php /** @var $row WPORG_GP_Translation_Events_Stats_Row */?>
+			<?php foreach ( $event_stats->rows() as $locale => $row ): ?>
+			<tr>
+				<td><?php echo esc_html( $locale ) ?></td>
+				<td><?php echo esc_html( $row->created ) ?></td>
+				<td><?php echo esc_html( $row->reviewed ) ?></td>
+				<td><?php echo esc_html( $row->users ) ?></td>
+			</tr>
+			<?php endforeach ?>
+			<tr class="event-details-stats-totals">
+				<td>Total</td>
+				<td><?php echo esc_html( $event_stats->totals()->created ) ?></td>
```

Comment:
This will cause a `Fatal error: Uncaught Error: Typed property WPORG_GP_Translation_Events_Event_Stats::$totals must not be accessed before initialization` if the table has no stats.

Reply:
Good catch, I had fixed it #35 but I've now cherry-picked the commit into here.